app/views/files/_form.html.haml

```diff
@@ -7,10 +7,7 @@
   = form.form_group t('.current_file') do
     - if @file.is_image?
       .file-image= image_tag @file.file.url
-    %ul
-      - FilesHelper::IMAGE_STYLES.each do |size|
-        %li
-          = link_to size.humanize, @file_presenter.full_path(style: size.to_sym), target: '_blank'
+    = link_to @file.file.path, @file.file.path, target: '_blank'
```

Comment:
I would use the full_path since the method was created mainly for this feature (or any sizes as well).